server/chat-plugins/poll.ts

```diff
@@ -350,6 +350,14 @@ export const commands: ChatCommands = {
 
 		clearqueue: 'deletequeue',
 		deletequeue(target, room, user, connection, cmd) {
+			const [slotString, roomid, update] = target.split(',').map(i => i.trim());
+			const targetRoom = roomid ? Rooms.search(roomid) : room;
+			if (!targetRoom) return this.errorReply(`Invalid room.`);
```

Comment:
Not necessary since requireRoom below does that.

Reply:
This check if targetRoom is falsy is still unnecessary - requireRoom does the same thing below.

Reply:
Yep, Done !

Reply:
Removing the lines, the deleteqeue does not work. @mia-pi-git

Reply:
How so?

Reply:
Oh. you removed too much of the code.
Restore the targetRoom stuff, you should only have removed the `if (!targetRoom)` line.

Reply:
```ts
if (roomid) {
const targetRoom = Rooms.search(target);
if (targetRoom) {
this.room = targetRoom;
}
}
room = this.requireRoom(); // etc
```

Reply:
Done, thx !